python_modules/libraries/dagster-dask/dagster_dask/config.py

```diff
@@ -42,6 +31,10 @@ def get_engine():
 
         return DaskEngine
 
+    @property
```

Comment:
I think you can avoid having to add this property by setting the default at init time with the check above:

```
cluster_type=check.opt_str_param(cluster_type, 'cluster_type', default='local')
```